pyquil/quilatom.py

```diff
@@ -139,12 +139,14 @@ def __eq__(self, other: object) -> bool:
     def register(cls, n: int) -> List["QubitPlaceholder"]:
         """Return a 'register' of ``n`` QubitPlaceholders.
 
+        >>> from pyquil import Program
+        >>> from pyquil.gates import H
+        >>> from pyquil.quil import address_qubits
+        >>> from pyquil.quilatom import QubitPlaceholder
         >>> qs = QubitPlaceholder.register(8) # a qubyte
         >>> prog = Program(H(q) for q in qs)
-        >>> address_qubits(prog).out()
-        H 0
-        H 1
-        ...
+        >>> address_qubits(prog).out()  # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
```

Comment:
```suggestion
        >>> address_qubits(prog).out()  # doctest: +NORMALIZE_WHITESPACE
```

Not used, right?

Also, if `NORMALIZE_WHITESPACE` is set as a default flag, do we need it here? No big deal either way.

Reply:
Yeah, not necessary, removed.